extlinks/common/swift.py

```diff
@@ -0,0 +1,91 @@
+import concurrent.futures
+import logging
+import os
+
+from typing import Iterable, List, Tuple
+
+import swiftclient
+
+import keystoneauth1.identity.v3 as identity
+import keystoneauth1.session as session
+
+logger = logging.getLogger("django")
+
+
+def swift_connection() -> swiftclient.Connection:
```

Comment:
Thank you for creating this file. For the sake of reusability, could we leverage these functions for use in the `linkevents_archive` command? https://github.com/WikipediaLibrary/externallinks/blob/master/extlinks/links/management/commands/linkevents_archive.py#L213